music21/harmony.py

```diff
@@ -1711,23 +1729,33 @@ def typeAlter(hD):
         for hD in chordStepModifications:
             if hD.modType == 'add':
                 typeAdd(hD)
-            elif hD.modType == 'subtract':
+            elif hD.modType == 'subtract' or hD.modType == 'omit':
                 typeSubtract(hD)
             elif hD.modType == 'alter':
                 typeAlter(hD)
 
         return tuple(pitches)
 
     def _getKindFromShortHand(self, sH):
+        all_abbr = [a for type in CHORD_TYPES.values() for a in type[1]]
```

Comment:
this is a substantial enough change it will need a battery of tests to make sure it works.  Thanks!

Reply:
(and documentation -- it should have had it originally; that was an oversight)